pkg/genericapiserver/options/server_run_options.go

```diff
@@ -383,6 +385,10 @@ func (s *ServerRunOptions) AddUniversalFlags(fs *pflag.FlagSet) {
 		"apis/<groupVersion>/<resource> can be used to turn on/off specific resources. api/all and "+
 		"api/legacy are special keys to control all and legacy api versions respectively.")
 
+	fs.Var(&s.FeatureConfig, "feature-config", ""+
+		"A set of key=value pairs that describe feature configuration that may be passed "+
+		"to apiserver. enable<FeatureName> key can be used to toggle alpha features. ")
```

Comment:
nit: why does this usage comment have a different format from the other feature-config flags?